bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java

```diff
@@ -187,50 +187,8 @@ public static String getIDFPath()
 	 */
 	public static String getIDFPythonEnvPath()
 	{
-		String idfPyEnvPath = new IDFEnvironmentVariables().getEnvValue(IDFEnvironmentVariables.IDF_PYTHON_ENV_PATH);
-		idfPyEnvPath = idfPyEnvPath.strip();
-		if (!StringUtil.isEmpty(idfPyEnvPath))
-		{
-
-			if (Platform.getOS().equals(Platform.OS_WIN32))
-			{
-				idfPyEnvPath = idfPyEnvPath + "/" + "Scripts"; //$NON-NLS-1$ //$NON-NLS-2$
-			}
-			else
-			{
-				idfPyEnvPath = idfPyEnvPath + "/" + "bin"; //$NON-NLS-1$ //$NON-NLS-2$
-			}
-			java.nio.file.Path commandPath = findCommand(IDFConstants.PYTHON_CMD, idfPyEnvPath);
-			if (commandPath != null)
-			{
-				return commandPath.toFile().getAbsolutePath();
-			}
-		}
-		return findCommandFromBuildEnvPath(IDFConstants.PYTHON_CMD);
-
-	}
-	
-	public static String getIDFPythonEnvPath(String idfPyEnvPath)
-	{
-		idfPyEnvPath = idfPyEnvPath.strip();
-		if (!StringUtil.isEmpty(idfPyEnvPath))
-		{
-
-			if (Platform.getOS().equals(Platform.OS_WIN32))
-			{
-				idfPyEnvPath = idfPyEnvPath + "/" + "Scripts"; //$NON-NLS-1$ //$NON-NLS-2$
-			}
-			else
-			{
-				idfPyEnvPath = idfPyEnvPath + "/" + "bin"; //$NON-NLS-1$ //$NON-NLS-2$
-			}
-			java.nio.file.Path commandPath = findCommand(IDFConstants.PYTHON_CMD, idfPyEnvPath);
-			if (commandPath != null)
-			{
-				return commandPath.toFile().getAbsolutePath();
-			}
-		}
-		return findCommandFromBuildEnvPath(IDFConstants.PYTHON_CMD);
+		IDFEnvironmentVariables idfEnvironmentVariables = new IDFEnvironmentVariables();
+		return idfEnvironmentVariables.getEnvValue(IDFEnvironmentVariables.PYTHON_EXE_PATH);
```

Comment:
Hmm...why do we an extra environment variable PYTHON_EXE_PATH with the v4.0.0, why can't we rely on IDF_PYTHON_ENV_PATH which we get it from the EIM? Do you have any specific case for this?